corehq/sql_db/routers.py

```diff
@@ -35,7 +36,7 @@ def allow_migrate(db, app_label):
 
     if app_label == PROXY_APP:
         return db == partition_config.get_proxy_db()
-    elif app_label == FORM_PROCESSOR_APP:
+    elif app_label in (FORM_PROCESSOR_APP, SCHEDULING_PARTITIONED_APP):
```

Comment:
the routing here is only used by the scheduling models for migrations and not for running queries, since the sharding is done in python. i actually created a separate app, `scheduling_partitioned`, apart from the `scheduling` app, to hold all scheduling models that can be partitioned. it's going to create the partitioned models in the proxy db (in case we ever want to use pl/proxy even though it's not using it right now) as well as reuse all the form processing dbs for the actual partitioned models that hold data.

@snopoke fyi